scripts/image-build/build-vyos-image

```diff
@@ -157,16 +179,19 @@ if __name__ == "__main__":
         else:
             parser.add_argument('--' + k, type=str, help=help_string, default=default_value_thunk())
```

Comment:
`str` object is not callable,  `default_value_thunk()`
should be `default_value_thunk`?

Reply:
The `default_value_thunk` is a function. A real example is `get_default_build_by` referenced in the `options` dict. Apparently PyCharm's parser isn't smart enough to understand that functions can be passed as arguments in Python.